src/autoschedulers/mullapudi2016/AutoSchedule.cpp

```diff
@@ -1256,7 +1288,7 @@ class GPUTilingDedup {
      * to map the tile orders. Here, we always cache the current dimension
      * order, and override the previous ones.
      */
-    void canReorder(const std::vector<VarOrRVar> &vars) {
+    void can_reorder(const std::vector<VarOrRVar> &vars) {
```

Comment:
Why is this called `can_reorder` when it seems to actually reorder? The `can` suggests it will test something and return a boolean, which it doesn't.

Also, above in `set_initial_order()`, maybe it's nice to set `is_initial_order` back to `true`?

Reply:
Suggestion of a new function name is welcome. Earlier I thought about `schedule_reorder`, but then the term `schedule` is already taken. I also considered `set_reorder` and `override_reorder` but it sounds misleading. What about `emplace_reorder`? Something that describes the idempotent nature of the action.